runtime/doc/builtin.txt

```diff
@@ -10242,7 +10243,8 @@ tempname()					*tempname()* *temp-file-name*
 		is different for at least 26 consecutive calls.  Example: >
 			:let tmpfile = tempname()
 			:exe "redir > " .. tmpfile
-<		For Unix, the file will be in a private directory |tempfile|.
+<		For Unix, the file will be in a private directory |tempfile|
```

Comment:
Somehow this comment was dropped in https://github.com/vim/vim/commit/5faeb60480c6efba5c0468c01275120b6ace5a09
Is this clear to everyone? I got caught off-guard once and kept wondering for a while.

Reply:
It didn't get dropped, I just added the missing trailing dot back and then there was no change anymore :)

Reply:
When looking at https://github.com/vim/vim/pull/14770/files there's also https://github.com/vim/vim/pull/14770/files#diff-ef8946d89919ad628b9ac65c49d3067e4e017b0980cac70eb25d832a59ef677dR10247 that was meant to warn against recursive deletion of everything inside the temp dir...

Reply:
hm, not sure how this got lost. Let me add it again.

Reply:
updated and further clarified here: 5cf5301e2847c5a1a380ae030c58825b5c6fe275